frontend/src/citizen-frontend/map/UnitDetailsPanel.tsx

```diff
@@ -166,7 +166,7 @@ export default React.memo(function UnitDetailsPanel({
             <div>
               {unit.roundTheClock ? t.map.shiftCareYes : t.map.shiftCareNo}
             </div>
-            {unit.url && (
+            {unit.url !== null && (
```

Comment:
I think the correct fix in a lot of cases (such as here) is to convert these into `!!unit.url` to keep functionality the same. Like here, I don't think it makes sense to have a homepage link that points to nowhere.